src/main/java/fmartin1/common/Endpoint.java

```diff
@@ -0,0 +1,43 @@
+package fmartin1.common;
+
+public class Endpoint {
```

Comment:
What's the purpose of this class?

Reply:
It's a standard container for endpoints in PokeApi, each endpoint returns the same attributes this class has, and it's already designed to work with pagination.